test/Engine/FFITarget/TestObjects.cs

```diff
@@ -457,4 +457,35 @@ public void Dispose()
         }
     }
 
+    public class TestRankReduce : IDisposable
```

Comment:
You don't need to make this class an `IDisposable`.

Reply:
did you intend to dispose this class?

Reply:
I only implemented `IDisposable` because the other test objects in this file did. I didn't really have any intentions for it.